### ISSUE TITLE
dehazing

### ISSUE THREAD

**Issue by EKChloe** [ZhendongWang6/Uformer#29]
do you think  retrain your model on dehazing work andk it will be useful？

**Comment by xin-fight** [comment]
> do you think retrain your model on dehazing work andk it will be useful？

hello, Have you experimented in the field of dehazing? How is the effect

**Comment by EKChloe** [comment]
No, I just found another model and used it to complete my undergraduate graduation project




------------------&nbsp;原始邮件&nbsp;------------------
发件人: ***@***.***&gt;; 
发送时间: 2022年3月5日(星期六) 上午9:20
收件人: ***@***.***&gt;; 
抄送: ***@***.***&gt;; ***@***.***&gt;; 
主题: Re: [ZhendongWang6/Uformer] dehazing (Issue #29)





  
do you think retrain your model on dehazing work andk it will be useful？
  
hello, Have you experimented in the field of dehazing? How is the effect
 
—
Reply to this email directly, view it on GitHub, or unsubscribe.
Triage notifications on the go with GitHub Mobile for iOS or Android.

**Comment by xin-fight** [comment]
> No, I just found another model and used it to complete my undergraduate graduation project
> […](#)
> ------------------&nbsp;原始邮件&nbsp;------------------ 发件人: ***@***.***&gt;; 发送时间: 2022年3月5日(星期六) 上午9:20 收件人: ***@***.***&gt;; 抄送: ***@***.***&gt;; ***@***.***&gt;; 主题: Re: [ZhendongWang6/Uformer] dehazing (Issue #29) do you think retrain your model on dehazing work andk it will be useful？ hello, Have you experimented in the field of dehazing? How is the effect — Reply to this email directly, view it on GitHub, or unsubscribe. Triage notifications on the go with GitHub Mobile for iOS or Android. You are receiving this because you authored the thread.Message ID: ***@***.***&gt;

What model did you use to pass the undergraduate project? I'm in the middle of my graduation project, too
